README.md

```diff
@@ -6,4 +6,14 @@ Installation
 
     $ go get github.com/cznic/crt
 
+Installation requires the following Go packages
```

Comment:
AFAICT, the installation, like in `go get` and `go install` does not require these tools. They're just called by the Makefile, which is supposed to be used only by a developer. Of course, documenting the list needed for running `make` is a good idea, please just adjust the meaning of the above sentence, thanks.